src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Rules/NewLineUserSettingFormattingRule.cs

```diff
@@ -159,6 +159,15 @@ private static bool IsControlBlock(SyntaxNode node)
                 }
             }
 
+            // * { - in property pattern clauses in pattern matching context
```

Comment:
💡 This block can be combined with the block on line 130 for `new {` and `with {`

Reply:
Wanted to ask if you prefer that 👍🏻